holmes/plugins/toolsets/investigator/core_investigation.py

```
@@ -74,7 +74,9 @@ def print_tasks_table(self, tasks):
 
         logging.info(separator)
 
-    def _invoke(self, params: Dict) -> StructuredToolResult:
+    def _invoke(
+        self, params: dict, user_approved: bool = False
```

Ruff (0.12.2)

Unused method argument: `user_approved`

(ARG002)